docs/preview/features/powershell/azure-api-management.md

```diff
@@ -20,6 +21,34 @@ To have access to the following features, you have to import the module:
 PS> Import-Module -Name Arcus.Scripting.ApiManagement
 ```
 
+## Backing up an API Management service
+
+Backs up an API Management service (with built-in storage context retrieval).
+
+| Parameter             | Mandatory | Description                                                                                                                                                      |
+| --------------------- | --------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `ResourceGroupName`   | yes       | The name of the of resource group under which the API Management deployment exists.                                                                              |
```

Comment:
The APIM service and the storage account can be located in separate resource groups. Currently, the implementation assumes those are located in the same resource group.
suggestion:
 - ResourceGroupName = APIM service resource group (as this module is APIM-centric)
 - StorageAccountResourceGroupName = Storage Account resource group

Reply:
Yep, done. Resolving.